ucf-document.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/*
+Plugin Name: UCF Document Plugin
+Description: Provides a custom post type and taxonomies for managing documents
+Version: 0.1.0
+Author: UCF Web Communications
+License: GPL3
+GitHub Plugin URI: UCF/UCF-Document-Plugin
+*/
+
+namespace UCFDocument;
+use UCFDocument\Document;
+
+if ( ! defined( 'WPINC' ) ) {
+    die;
+}
+
+/**
+ * Constants
+ */
+define( 'UCF_DOCUMENT__PLUGIN_FILE', __FILE__ );
+define( 'UCF_DOCUMENT__PLUGIN_PATH', dirname( __FILE__ ) );
+
+/**
+ * Includes
+ */
+require_once 'includes/document-post-type.php';
+require_once 'includes/utilities.php';
+
+/**
+ * Activation/Deactivation Hooks
+ */
+
+/**
+ * Function that runs when the plugin
+ * is activated.
+ * @author Jim Barnes
+ * @since 1.0.0
+ * @return void
+ */
+function activate() {
+	Document\register();
```

Comment:
This is redundant now that the Document registration function is called in the constructor of that class

Reply:
`register` has to be called before the `flush_rewrite_rules` function is called, so a manual call has to be done. Updated to reflect the register function is in a class now.